spec/board_spec.rb

```diff
@@ -56,6 +56,15 @@
       expect(board.valid_placement?(submarine, ["A2", "A3", "A4"])).to eq(false)
     end
 
+    it 'must be consecutive' do 
+      board = Board.new
+      cruiser = Ship.new("Cruiser", 3)
+
+      expect(board.valid_placement?(cruiser, ["A1", "A2", "A4"])).to eq(false)
+    end 
+ #### test #cell_compare? more thoroughly
+
```

Comment:
I think our next step is to write more tests to confirm the functionality of this method. Should we make a separate describe "#cell_compare"?